packages/core/src/components/Tabs/Tab/Tab.styled.tsx

```diff
@@ -1,32 +1,111 @@
 import { SvgIcon } from '@medly-components/icons';
-import { Theme } from '@medly-components/theme';
+import { Theme } from '@medly-components/theme/src';
 import { css, styled } from '@medly-components/utils';
-import { StyledProps } from './types';
+import { TextStyled } from '../../Text/Text.styled';
+import { TabBackground, TabSize, TabStyle } from '../types';
+import { StyledProps, WrapperProps } from './types';
 
-const activeStyle = ({ tabs }: Theme) =>
-    css`
-        color: ${tabs.activeTabTextColor};
-        border-bottom: 2px solid ${tabs.activeTabTextColor};
+const getTabSpacing = (hasIcon = false) => {
+    const verticalSpacing = 15;
+
+    if (hasIcon) {
+        return verticalSpacing - 1;
+    }
+    return verticalSpacing;
+};
+
+const activeStyle = (theme: Theme, active: boolean, tabSize: TabSize, tabStyle: TabStyle, tabBackground: TabBackground) => {
```

Comment:
we could use something like this----> `const activeStyle = ({theme:, active, tabSize, tabStyle, tabBackground}: WrapperProps)`. Styled component will automatically pass the prop object.